scripts/cfos/tests/README.md

```diff
@@ -1,18 +1,24 @@
 ## Test schema file setup ##
-You can either:
-- replace the contents of 'scripts/cfos/dataset_tables_schema.json' file with the contents of the 'scripts/cfos/tests/test_schema.json' file temporarily OR
-- change the code in make_dataset_data_tables.py so that schema_json points to the test schema file instead of the real one
+You can run the code using the test schema by adding the parameter "-t True" to run in testing mode (examples shown below in How to Run section)
 
 ## How to Run ##
 ### test fail ### 
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 ### test pass ###
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 
 ## Validating outcomes ##
 Check the validation_erros.csv and make sure expected outcomes are met. 
 General expected outcomes are explained above the column ID in the Fail.xlsx file. 
 
+The code can fail in the following ways:
+- **primary key column contains non-unique values**: 'validation_error':{row: 6, column: "id_unique"}: "X1234B" contains values that are not unique
+- **value doesn't match an expected pattern** : 'validation_error':{row: 6, column: "file_path_explicit_pattern"}: "gs://456" does not match the pattern "^fs://"
```

Comment:
minor, but it's confusing to me that this example uses a different string match than you'd likely expect - like I'd expect the pattern to match to be "^gs://" and the example non-matching string to be "fs://456". for a second i thought i was misunderstanding the pattern formatting.

Reply:
It doesn't really matter either way - in this case I pulled the error for the case where I over-wrote the default pattern matching and was testing to make sure it applied that instead of the default. Therefore "^gs" should fail because it expects something different. 

I went ahead and changed the values - the purpose of this isn't that this value will always be the required pattern to match, but rather that this is the format of the error message when a pattern is not matched.